gherkin/GHERKIN_MARKDOWN.md

```diff
@@ -0,0 +1,42 @@
+# Gherkin Markdown
+
+Gherkin Markdown lets you write Cucumber scenarios in Markdown. Plain and simple.
+
+Gherkin Markdown is not a Markdown dialect per se, but rather a set of rules for
+parsing scenarios from any Markdown document.
+
+The parsing rules are as follows:
+
+- `#` - Feature
```

Comment:
Does this only support the `#` syntax for specifiying H1? or also the underline syntax?

e.g.

```
An H1
====
```

Would that also be matched as a feature?

Reply:
Not at this time - implementing support for the underline syntax would be too hard for the current line-oriented parser.